graphql-spqr-spring-boot-autoconfigure/src/main/java/io/leangen/graphql/spqr/spring/modules/data/DefaultValueSchemaTransformer.java

```diff
@@ -13,7 +13,7 @@ public interface DefaultValueSchemaTransformer extends SchemaTransformer {
 
     @Override
     default GraphQLArgument transformArgument(GraphQLArgument argument, OperationArgument operationArgument, OperationMapper operationMapper, BuildContext buildContext) {
-        if (supports(operationArgument.getJavaType()) && !(argument.getType() instanceof GraphQLNonNull) && argument.getDefaultValue() == null) {
+        if (supports(operationArgument.getJavaType()) && !(argument.getType() instanceof GraphQLNonNull) && argument.getArgumentDefaultValue() == null) {
```

Comment:
should it not be `argument.getArgumentDefaultValue().getValue()`?

Reply:
Thank you Joachim for looking into this!

Most likely, but as I said, I don't know the underlying lib and have not the cycles to test that part. Happy to change as needed.